src/DrDotnet.Profilers/session/session_info.rs

```diff
@@ -0,0 +1,78 @@
+use std::path::{PathBuf, Path};
+use std::io::BufWriter;
+use std::fs::File;
+use std::io::Write;
+use std::str::FromStr;
+
+use crate::rust_protobuf_protos::interop::*;
+
+use crate::session::Report;
+
+impl SessionInfo {
+
+    // Returns a Session from its UID and ProfilerData.
+    // If the Session report is not present on the disk, it will be written at the same time.
+    pub fn create_session_json(&self) {
+
+        // Serialize to JSON
+        let json = protobuf_json_mapping::print_to_string(self).unwrap();
```

Comment:
There is an issue here, we are serializing to JSON a `SessionInfo` which has the following fields:
```rust
pub struct SessionInfo {
    // message fields
    // @@protoc_insertion_point(field:SessionInfo.uuid)
    pub uuid: ::std::string::String,
    // @@protoc_insertion_point(field:SessionInfo.parameters)
    pub parameters: ::std::vec::Vec<ProfilerParameter>,
    // special fields
    // @@protoc_insertion_point(special_field:SessionInfo.special_fields)
    pub special_fields: ::protobuf::SpecialFields,
}
```
but the C# part is expecting to deserialize a `Session` which has the following fields:
```C#
public class Session
{
    public Guid SessionId { get; set; }

    public string ProcessName { get; set; }

    public DateTime Timestamp { get; set; }

    public ProfilerInfo Profiler { get; set; }
(...)
```

Reply:
Ahh yes I'll fix this